### DISCUSSION TITLE
Does videojs work on Tizen and WebOS? (Smart TV)

### DISCUSSION THREAD

**Discussion by kyojunlee** [videojs/video.js#7597] [Q&A]
Hi I would like to know the followings..

1. Videojs compatibility with Tizen and WebOS
2. Can videojs play HEVC(H.265 - 8K video) Playready DRM protected contents on Tizen and WebOS?

**Comment by ParasPidurkar** [comment]
@kyojunlee  though in the videojs 8.0 there is a support for Tizen (2018+ models ) and Webos 4 .I have observed that there is a issue in playing the HD and UHD  H264 streams though HEVC works fine on tizen  .I was able to play the widevine DRM streams not sure about playready though .

**Comment by sarah-harissa** [comment]
Hi did you implement  quality selector in videojs for tizen and webos lg?
